core/src/main/java/com/yahoo/oak/BasicChunk.java

```diff
@@ -19,6 +19,7 @@
         RELEASED
     }
 
+    static final int NONE_NEXT = EntryArray.INVALID_ENTRY_INDEX;
```

Comment:
I see why `NONE_NEXT` was moved here. Because `BasicChunkIter.next(ThreadContext ctx)` should return it when there is no more entries left to return.

However, `NONE_NEXT` is more part of OrderedChunk, because there we have linked list, with "nexts" in each entry. `NONE_NEXT` is not used in HashChunk. Can something else be invented?

Reply:
I guess I can just use EntryArray.INVALID_ENTRY_INDEX. Seems to fit the purpose